### PR TITLE
Add postgres db resource

### PR DESCRIPTION
**Pull request by VILLAN3LL3**

### CODE REVIEW

File: terraform/azure/variables.tf [hunk 1]
```diff
@@ -16,6 +16,24 @@ variable "club" {
...
+variable "dbconfig" {
+  description = "Settings for the postgresql db server"
+  type = object({
+    username     = string // ToDo: Take from key vault
+    password     = string // ToDo: Take from key vault
+    sku          = string
+    storage      = number
+    databaseName = string
+  })
+  default = {
```
Comment (balsmn) [inline terraform/azure/variables.tf]: This default won't work. You have to move it to locals and use the "default" function.

**Review by balsmn — DISMISSED** [review]
I will approve for now. After you correct the one comment, you can merge it. Thanks

File: terraform/azure/variables.tf (hunk 1 above)
Comment (VILLAN3LL3) [reply terraform/azure/variables.tf]: I'm not sure if I got your point. I implemented the default in the same way as I did for the backendconfig, see below. What is the difference between those two?
What do you mean with "locals"?

Comment (balsmn) [reply terraform/azure/variables.tf]: Yes, but we should change it, as the default value don't work as expected after 0.12.

I meant something like this https://www.terraform.io/docs/language/functions/defaults.html

Comment (VILLAN3LL3) [reply terraform/azure/variables.tf]: @balsmn I've now replaced the defaults with locals, but neither the documentation recognizes the defaults now nor the tests pass. Before, everything was ok.

**Review by balsmn — APPROVED** [review]
LGTM.

**Pull request merged**